.prettierrc

```diff
@@ -2,5 +2,14 @@
   "trailingComma": "es5",
   "tabWidth": 2,
   "semi": true,
-  "singleQuote": true
+  "singleQuote": true,
+  "printWidth": 80,
+  "editor.formatOnSave": true,
```

Comment:
I think the `editor.*` `eslint.*`, and `prettier.*` options here are actually meant to live in VS Code workspace or user settings.

Reply:
Reverted.